parlai/core/torch_agent.py

```diff
@@ -1323,12 +1344,8 @@ def batchify(self, obs_batch, sort=False):
             labels = [ex.get(field + '_choice') for ex in exs]
             y_lens = [y.shape[0] for y in label_vecs]
 
-            ys, y_lens = padded_tensor(
-                label_vecs,
-                self.NULL_IDX,
-                self.use_cuda,
-                fp16friendly=self.opt.get('fp16'),
-            )
+            ys, y_lens = self._pad_tensor(label_vecs)
```

Comment:
ah this is gonna conflict hard with @klshuster

Reply:
uh oh...really?

Reply:
Not too bad, he's just gonna have a small merge conflict.

Reply:
yeah a little conflict but nothing major